github/actions_workflows.go

```diff
@@ -47,6 +47,12 @@ type WorkflowBill struct {
 	TotalMS *int64 `json:"total_ms,omitempty"`
 }
 
+// CreateWorkflowDispatchEventRequest represents a request to create a workflow dispatch event
+type CreateWorkflowDispatchEventRequest struct {
+	// Ref is required when creating a workflow dispatch event
+	Ref string `json:"ref,omitempty"`
```

Comment:
Let's remove `omitempty` from this parameter since it is required.

```suggestion
	Ref string `json:"ref"`
```